src/styles/components/buttons/styles.less

```diff
@@ -49,48 +57,45 @@ components/buttons.less
       &.button-fixed-width-small {
         width: @base-spacing-unit * 3;
```

Comment:
Seems like this fixed width could create some issues on certain screen-sizes vs. something more fluid.